assets/src/edit-story/components/movable/index.js

```diff
@@ -130,8 +130,8 @@ function Movable( {
 			} }
 			origin={ false }
 			pinchable={ true }
-			keepRatio={ 'image' === selectedElement.type } // @†odo Even image doesn't always keep ratio, consider moving to element's model.
-			renderDirections={ 'image' === selectedElement.type ? CORNER_HANDLES : ALL_HANDLES }
+			keepRatio={ selectedElement.keepRatio || false }
```

Comment:
Any reason for not using just `selectedElement.keepRatio` here, or its boolean value? Readability?

Reply:
For now `keepRatio` is optional since we weren't sure if we wanted it in other elements. Do you think we should just make it required?

Reply:
This part is reimplemented.